accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -730,4 +733,5 @@ In this case, most of the parameters are *unnecessary*, and a tear-off expressio
 * 2.12: Mention abstract classes.
 * 2.13: Add `is` and `as` disambiguation tokens.
 * 2.14: Remove many disambiguation tokens. Allow instantiating function *objects* and *callable objects*. Mention forwarding constructors from mixin applications.
-* 2.15: Add section about constants and specify new rules about potentially constant and constant expressions of the form `e<T1..Tk>`.
+* 2.15: Add section about constants and specify new rules about potentially constant and constant expressions of the form <code>e\<T<sub>1</sub>..T<sub>k</sub>></code>.
```

Comment:
(Want to be really fancy, change "e", "|T" and "k" to "\*e\*", "\*T\*" and "\*k\*" in the code to highlight that they're (meta-)variables, not literals).

Reply:
Done. That was actually a rather widespread operation, but it does indicate the meaning more explicitly.